app/src/androidTest/java/com/android/sample/createRecipe/RecipeNameScreenTest.kt

```diff
@@ -31,7 +34,8 @@ class RecipeNameScreenTest {
     // Mock Firestore and Repository for use in ViewModel
     val firestore = mockk<FirebaseFirestore>(relaxed = true)
     val repository = FirestoreRecipesRepository(firestore)
-    createRecipeViewModel = spyk(CreateRecipeViewModel(repository)) // Spy to verify calls
+    repoImg = mockk(relaxed = true)
+    createRecipeViewModel = spyk(CreateRecipeViewModel(repository, repoImg)) // Spy to verify calls
 
     // Set the content for testing
 
```

Comment:
The `tearDown()` method currently does nothing, as there are no resources being released or clean-up actions required. Since it's not serving any purpose, I suggest deleting it to keep the code cleaner and more concise.